aioshelly/ble/__init__.py

```diff
@@ -85,6 +87,8 @@ def create_scanner(
             can_connect=lambda: False,
         ),
         False,
+        requested_mode=requested_mode,
+        current_mode=current_mode,
```

Comment:
Shelly will always be in sync so requested_mode and current_mode are always the same